Microsoft.Toolkit.Uwp/Helpers/BackgroundTaskHelper.cs

```diff
@@ -0,0 +1,107 @@
+﻿// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.ApplicationModel.Background;
+
+namespace Microsoft.Toolkit.Uwp.Helpers
+{
+    /// <summary>
+    /// This class provides static helper methods for background task.
+    /// </summary>
+    public static class BackgroundTaskHelper
+    {
+        /// <summary>
+        /// Check if a background task has been registered with the application or not.
```

Comment:
"or not" at the end of the summary feels awkward. Suggest removing it? Also, just remove "with the application" too? That's kind of implied. And should be "is registered", not "has been", since "has been" implies that it was simply registered at some point in time, not that it's actively registered.

So I would suggest: "Check if a background task is registered."